index.html

```diff
@@ -81,11 +83,25 @@ <h2 class="topTitle">Top three movies</h2>
             </section>
          </section>
          <section class="lowerSection">
-            <figure class="uppKommingMovies">
+            <figure class="upComingMovies">
                <!--up coming movies here-->
             </figure>
-            <div class="uppKommingEvents">
+            <div class="upComingEvents">
                <!--All the events here-->
+               <hr class="eventLine">
+               <h2>Kommande evenemang</h2>
+               <div class="cardsContainer">
+                  <div class="eventCard">
+                     <img class="eventImg" src="Pictures/event1.png" alt="Event image not found">
+                     <p class="eventInfo">Here goes the information about event</p>
+                     <button class="eventBtn">Buy Ticket</button>
```

Comment:
Instead of a button with color **blue**. Maybe we should follow a design with an icon and color scheme that follows a more easy to the eyes color scheme. (like on Nordiska film)<br>**See the img for example how I mean (where the icon is):**
![Skärmavbild 2024-12-30 kl  10 36 05](https://github.com/user-attachments/assets/c975d5a1-8176-403c-8850-76b2c9dca72c)